app/admin/check_ins.rb

```diff
@@ -0,0 +1,21 @@
+ActiveAdmin.register CheckIn do
+  menu parent: 'Event'
+
+  index do
+    column 'Event', sortable: :event_name do |check_in|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.